docs/advocate-kit/why.md

```diff
@@ -0,0 +1,40 @@
+# Why did we create these Principles?
+
+This page describes the reasoning, intentions, and thoughts behind the Principles of Authentic Participation.
+While it does not talk about the Principles specifically, it adds background context and history for how the initial work on the Principles began.
+
+
+## History
+
+The Principles of Authentic Participation were derived at the [Sustain Summit 2020](https://blog.jwf.io/2020/02/sustain-oss-2020-quick-rewind/) event on 30 January 2020 in Brussels, Belgium.
+There, [Duane O'Brien](https://github.com/DuaneOBrien) and others facilitated discussion groups loosely focused on corporate accountability in the context of open source.
+From these discussion groups, the following four goals emerged based on discussion topics from the day:
+
+1. Set and publish a goal for open source contribution relative to value capture
+1. Adhere to the Principles of Authentic Participation
+1. Publish documentation of open source policies, processes, and project governance
+1. Well-defined reporting process that is publicly available
```

Comment:
Should these be numbered in order? (1, 2, 3, 4). They're all 1s currently

Reply:
It looks this way in the source text, but this is a Markdown ordered list. When rendered, it will automatically iterate based on the listed order.